src/MessagePack.GeneratorCore/CodeGenerator.cs

```diff
@@ -117,40 +115,33 @@ public static string GenerateSingleFileSync(string resolverName, string namespac
                 {
                     var (nameSpace, isStringKey) = x.Key;
                     var objectSerializationInfos = x.ToArray();
-                    var template = isStringKey ? new StringKeyFormatterTemplate() : (IFormatterTemplate)new FormatterTemplate();
-
-                    template.Namespace = namespaceDot + "Formatters" + (nameSpace is null ? string.Empty : "." + nameSpace);
-                    template.ObjectSerializationInfos = objectSerializationInfos;
-
+                    var ns = namespaceDot + "Formatters" + (nameSpace is null ? string.Empty : "." + nameSpace);
+                    var template = isStringKey ? new StringKeyFormatterTemplate(ns, objectSerializationInfos) : (IFormatterTemplate)new FormatterTemplate(ns, objectSerializationInfos);
                     return template;
                 })
                 .ToArray();
 
+            string GetNameSpace<T>(IGrouping<string?, T> x)
```

Comment:
nit: `Namespace` is one word. No need to capitalize the `S`

Reply:
Thank you for the review!
I fixed it.